index.html

```diff
@@ -161,11 +161,39 @@ <h4>Experimental content creation feature that allows users to add to an existin
           <li>css</li>
           <li>javascript</li>
         </ul>
-        <a href="https://twitter.com/" class="button">See Project</a>
+        <a href="#" class="button">See Project</a>
       </div>
      
     </div>
   </section>
+
+  <!-- Mobile modal -->
+
+  <section class="popup">
```

Comment:
[ OPTIONAL ]Pop-up does not show buttons on small and medium viewports. KIndly see the screenshot below and fix the issue.
![pop-up-small](https://user-images.githubusercontent.com/35504149/132666840-65434fc3-764e-4888-8ed4-5f46c033f4ff.png)